pkg/claim/claim.go

```diff
@@ -0,0 +1,140 @@
+package claim
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"os"
+	"time"
+
+	ljsonrpc "github.com/lbryio/lbry.go/v2/extras/jsonrpc"
+
+	"go.uber.org/zap"
+)
+
+const (
+	lbrytvAPI = "https://api.lbry.tv/api/v1/proxy"
+	cdnServer = "https://cdn.lbryplayer.xyz/api/v3/streams"
+)
+
+var (
+	lbrytvClient   = ljsonrpc.NewClient(lbrytvAPI)
+	downloadClient = &http.Client{
```

Comment:
i recommend making an interface like 

```
type HttpRequester interface {
	Do(req *http.Request) (res *http.Response, err error)
}
```

then making `downloadClient` that type. not necessary in this code, but useful for testing